servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -512,6 +526,21 @@ public Builder cancellationIsError(final boolean cancellationIsError) {
             return this;
         }
 
+        /**
+         * Set the penalty factory to apply to outstanding requests.
+         * The EWMA penalty to apply to endpoints when there are outstanding requests. By penalizing endpoints with
+         * outstanding load the traffic distribution will be smoother for algorithms that consider load metrics.
+         * Higher penalties will favor a more even request distribution while lower penalties will bias traffic toward
+         * endpoints with better performance.
```

Comment:
Consider clarifying expected ranges. Higher/lower is not enough bcz ppl may wonder what is the behavior for negative values and what value to use to effectively "disable" penalty.